recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
-{% set version = "4.9.8" %}
-{% set build = 1 %}
+{% set version = "4.9.9-beta01" %}
+{% set build = 0 %}
 
 package:
   name: nco
   version: {{ version }}
 
 source:
   url: https://github.com/nco/nco/archive/{{ version }}.tar.gz
```

Comment:
```suggestion
  url: https://github.com/nco/nco/archive/4.9.9-beta01.tar.gz
```

Reply:
Sounds good. If this does not work I can change the format of the NCO version names to eliminate hyphens...